frontend/src/screens/wallet/index.tsx

```diff
@@ -24,14 +26,34 @@ function Wallet() {
   const { data: info, hasChannelManagement } = useInfo();
   const { data: balances } = useBalances();
   const { data: channels } = useChannels();
+  const navigate = useNavigate();
 
   if (!info || !balances) {
     return <Loading />;
   }
 
   return (
     <>
-      <AppHeader title="Wallet" description="" />
+      <AppHeader
+        title="Wallet"
+        description=""
+        contentRight={
+          <>
+            <ResponsiveButton
+              icon={ArrowDownIcon}
+              text="Receive"
+              size="lg"
+              onClick={() => navigate("/wallet/receive")}
```

Comment:
can't these be normal links? let's avoid using navigate for standard navigation as it breaks browser UX